common/djangoapps/dark_lang/middleware.py

```diff
@@ -158,8 +129,3 @@ def _activate_preview_language(self, request):
 
```

Comment:
Nit: is generally preferable for code readability and debugging to have only 1 return statement/flow in a method.  Since this is a small enough method with few paths, remove this `return` statement and negate the `if` clause to update the session when `preview_lang` is set.